### DISCUSSION TITLE
File Upload Fails at 100 percent

### DISCUSSION THREAD

**Discussion by TheKevster101** [iterate-ch/cyberduck#14930] [Q&A]
This ticket system is to report bugs and feature requests. For support, visit the [help page](https://cyberduck.io/help/) first.

 * If you have trouble connecting to a server or your login credentials are not valid, try to resolve the issue with the assistance of your hosting service provider. Any such ticket will be closed with a resolution of _thirdparty_.

**Describe the bug**
File Upload reaches 100% completion and uploading stops.  The STOP button doesn't work (on any uploads). When attempting to close Cyberduck, I get an error saying "Transfer in progress - There are files currently being transferred. Quit Anyway?" After force quit and restart, the files show as Transfer Incomplete with 0B of 0B.  I have left one case running for over 24 hours and it did not complete.

**To Reproduce**
1. Drag file to the desired folder on the cyberduck interface
2. Wait for upload to complete
3. See that it isn't complete but reads 100% and isn't uploading anymore. 
4. Try again...

**Expected behavior**

File Upload reaches 100% completion and uploading stops.  The STOP button doesn't work (on any uploads). When attempting to close Cyberduck, I get an error saying "Transfer in progress - There are files currently being transferred. Quit Anyway?" After force quit and restart, the files show as Transfer Incomplete with 0B of 0B.  I have left one case running for over 24 hours and it did not complete.

**Screenshots**

<img width="564" alt="Screen Shot 2023-07-23 at 8 16 31 PM" src="https://github.com/iterate-ch/cyberduck/assets/32725577/888b9e4c-2781-4f0a-a666-f444ce6cfe2e">

<img width="554" alt="Screen Shot 2023-07-23 at 8 21 17 PM" src="https://github.com/iterate-ch/cyberduck/assets/32725577/f22f4a2d-b74a-4ace-8e48-25ce5f7be985">

**Desktop (please complete the following information):**
 - OS: macOS Monterey
 - Version 12.6.7

**Log Files**
Please include the [log file](https://docs.cyberduck.io/cyberduck/support/#logging-output). You can reach the logging output in _Preferences → Connection_.
[cyberduck.log](https://github.com/iterate-ch/cyberduck/files/12140335/cyberduck.log)

**Additional context**

Thank you for any and all help!!!

**Comment by dkocher** [comment]
The error message from S3 is `Your socket connection to the server was not read from or written to within the timeout period. Idle connections will be closed.` [^1] 


[^1]: 
```
<?xml version="1.0" encoding="UTF-8"?>
<Error>
    <Code>RequestTimeout</Code>
    <Message>Your socket connection to the server was not read from or written to within the timeout period. Idle connections will be closed.</Message>
    <RequestId>BRT7TG6EA5539GJS</RequestId>
    <HostId>5b2IQQNMLVviGGL/4e91FpbeY9/Z/pUqglHvH6e97J5CzEqXjKMnX3MDNIRgZmiRSyr1B02+h+U=</HostId>
</Error>
```
